Cargo.toml

```diff
@@ -49,7 +49,7 @@ serde_plain = "0.3.0"
 serde_urlencoded = "0.5.1"
 
 # Code gen
-tower-web-macros = { version = "0.2.1", path = "tower-web-macros" }
+tower-web-macros = { version = "0.2.2", path = "tower-web-macros" }
```

Comment:
Is this version bump required?

Reply:
Umm, I'm not sure, but it seemed a good idea to keep the `tower-web` and `tower-web-macros` in lockstep, at least because #97 added a `tower-web` test that requires the corresponding `tower-web-macros` changes.

Would you rather I undid that?

Reply:
Unsure.

Usually, I only bump the minimum dependency version when I actually require that version to compile (due to new APIs). However, tower-web-macros is a bit special...

Reply:
Does it matter (with the two crates being in the same repository? Applications using `tower-web` don't need to depend on `tower-web-macros` anyway.

And having the tests pass sounds like a good thing :smile:.

Reply:
Sorry, I didn't realize the two versions weren't in lock-step. Fixed.